net.jenyay.Outwiker.yml

```diff
@@ -0,0 +1,36 @@
+app-id: net.jenyay.Outwiker
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: outwiker
+rename-icon: net.jenyay.Outwiker
+finish-args:
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
+modules:
+  - name: outwiker
+    buildsystem: simple
+    no-autogen: true
+    sources:
+      - type: git
+        url: https://github.com/Jenyay/outwiker
+        commit: 92735a55330df4350f02d8d60075ebb9e3efc47b
+        tag: flatpak-2
+    build-options:
+      build-args:
+        - --share=network
+    build-commands:
+      - "pip3 install --prefix=${FLATPAK_DEST} -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-18.04 wxPython==4.1.1"
+      - "pip3 install --prefix=${FLATPAK_DEST} Pillow==8.1.2 cyhunspell==2.0.2 idna>=2.10 pyparsing==2.4.7 psutil==5.8.0"
+      - make -f Makefile.flatpak install
+  - name: libjpeg
+    buildsystem: autotools
```

Comment:
libjpeg is in the runtime, you don't need to build it.

Reply:
wxPython requires libjpeg 8. Without this library, an error occurs at startup:
```
ImportError: libjpeg.so.8: cannot open shared object file: No such file or directory
```

Reply:
that's because you want to use binaries built for a 3 year old ubuntu....

Reply:
Which Ubuntu version is actual for flatpak?

Reply:
None.

Reply:
Ideally you should build it from source. You can find examples at https://github.com/search?q=org%3Aflathub+wxpython&type=code